src/zoid/checkout/component.jsx

```diff
@@ -163,6 +163,12 @@ export function getCheckoutComponent() : CheckoutComponent {
                     required:   false
                 },
 
+                apmBrandedStandaloneButton: {
```

Comment:
Can we just go with `branded`?

Reply:
`branded` seems to be an existing param https://github.com/paypal/paypal-smart-payment-buttons/pull/137/files#diff-e3a4efeee1834a0c5a8750c378aef4deb5d1a4d1638b72565c8b91759ab23211R167

Reply:
Yes. We can use this param for the same purpose. It's designed to differentiate between branded and unbranded buttons.